fv/connectivity/conncheck.go

```diff
@@ -56,24 +56,25 @@ type Checker struct {
 }
 
 func (c *Checker) ExpectSome(from ConnectionSource, to ConnectionTarget, explicitPort ...uint16) {
-	c.expect(true, from, to, explicitPort)
+	c.expect(true, from, to, ExpectWithPorts(explicitPort...))
 }
 
 func (c *Checker) ExpectSNAT(from ConnectionSource, srcIP string, to ConnectionTarget, explicitPort ...uint16) {
 	c.CheckSNAT = true
-	c.expect(true, from, to, explicitPort, ExpectWithSrcIPs(srcIP))
+	c.expect(true, from, to, ExpectWithPorts(explicitPort...), ExpectWithSrcIPs(srcIP))
 }
 
 func (c *Checker) ExpectNone(from ConnectionSource, to ConnectionTarget, explicitPort ...uint16) {
-	c.expect(false, from, to, explicitPort)
+	c.expect(false, from, to, ExpectWithPorts(explicitPort...))
 }
 
 // ExpectConnectivity asserts existing connectivity between a ConnectionSource
 // and ConnectionTarget with details configurable with ExpectationOption(s).
 // This is a super set of ExpectSome()
-func (c *Checker) ExpectConnectivity(from ConnectionSource, to ConnectionTarget,
-	ports []uint16, opts ...ExpectationOption) {
-	c.expect(true, from, to, ports, opts...)
+func (c *Checker) ExpectConnectivity(connectivity bool,
```

Comment:
How about dropping `Connectivity` since it's implicit given this is a method of connectivity checker?

I'm not so keen on a bare true/false argument, how about `cc.Expect(conncheck.Blocked|Allowed, ...)`  (but then the conncheck prefix is a bit verbose)